src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -160,6 +178,24 @@ void AlertManeuverRequest::on_event(const event_engine::Event& event) {
       tts_speak_result_code_ = static_cast<mobile_apis::Result::eType>(
           message[strings::params][hmi_response::code].asInt());
 
+      tts_result = MessageHelper::MobileToHMIResult(tts_speak_result_code_);
+
+      is_tts_ok = Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
+          tts_result,
+          hmi_apis::Common_Result::SUCCESS,
+          hmi_apis::Common_Result::UNSUPPORTED_RESOURCE,
+          hmi_apis::Common_Result::WARNINGS,
+          hmi_apis::Common_Result::INVALID_ENUM);
+
+      const bool is_tts_success =
+          Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
+              tts_result, hmi_apis::Common_Result::SUCCESS);
```

Comment:
I think it is excessive to use `Compare` for only one parameter. It is better to use direc equal operator